admin_guide/backup_restore.adoc

```diff
@@ -707,15 +707,18 @@ Copy the three environment variables in the etcdctl member add output. They will
 * ETCD_INITIAL_CLUSTER
 * ETCD_INITIAL_CLUSTER_STATE
 +
-Replace the IP address with the "NEW_ETCD" value for:
+.. Replace the IP address with the "NEW_ETCD" value for:
 +
 * ETCD_LISTEN_PEER_URLS
 * ETCD_LISTEN_CLIENT_URLS
 * ETCD_INITIAL_ADVERTISE_PEER_URLS
 * ETCD_ADVERTISE_CLIENT_URLS
 +
-For replacing failed members, you will need to remove the failed hosts from the
-etcd configuration.
+.. For replacing failed members, replace the failed hosts with the new hosts.
```

Comment:
Just as Daniel suggested in the bug, I'll also prefer it could be something like:
For replacing failed members, you will need to replace the failed etcd hosts with the new etcd hosts from the etcd configuration, also on the other etcd nodes.

Reply:
@gpei, is this better?

Reply:
Yes, it's much clear now :+1: